sdk/docs/man/eglCreatePlatformPixmapSurface.xml

```diff
@@ -163,24 +163,17 @@
         </para>
     </refsect1>
     <refsect1 xml:id="notes"><title>Notes</title>
+        <para>
+            <code class="function">eglCreatePlatformPixmapSurface</code> is supported only if
+            the EGL version is 1.5 or greater.
+        </para>
         <para>
             The <constant>EGL_MATCH_NATIVE_PIXMAP</constant> attribute
             of
             <citerefentry><refentrytitle>eglChooseConfig</refentrytitle></citerefentry>
             can be used to select a frame buffer configuration matching
             a specified native pixmap.
         </para>
-        <para>
```

Comment:
From the spec appendix and egl.h versioning. History looks to be. COLORSPACE was added with EGL 1.2. With EGL 1.3 the VG_COLORSPACE was added. Then in EGL 1.5 GL_COLORSPACE was added to core.
GL_COLORSPACE was/is an extension prior to EGL 1.5.
So, maybe not delete but modify these notes? Jon, what do you recommend?

Reply:
Agreed. I'm not clear why these notes should be removed. It would be nice to clarify them.

Reply:
So for first part: only if EGL 1.5 or extension EGL_KHR_gl_colorspace is supported.

Second part could stay.

Reply:
Hm. I removed the notes because I didn't think they're relevant to users of `eglCreatePlatform{Pixmap,Window}Surface` - the entry point only exists in EGL 1.5, users are guaranteed to be able to use those attributes, since they're all core in 1.5?

I'm happy to add them back if that's desired, though.

Reply:
OK. Yes. Since this is man pages, it doesn't take in to account extensions. So you are right, its an entry point only dependent on EGL 1.5.